web-app/src/main/webapp/script.js

```diff
@@ -59,8 +61,25 @@ function showMessageOnInfoWindow(message, position, map, infoWindow) {
 window.onload = () => {
   document.getElementById("form-container").style.display = "none";
   document.getElementById('report-button').addEventListener('click', showReportForm);
+  loadPoliceReports();
 };
 
 function showReportForm() {
   document.getElementById("form-container").style.display = "block";
 }
+
+async function loadPoliceReports() {
+  const FILE_NAMES = ['2019_12_london', '2020_01_london', '2020_02_london', '2020_03_london', '2020_04_london', '2020_05_london']
```

Comment:
Is there a way to do this without hardcoding names? I guess it isn't too much of a hassle since we only have to change two strings once a month.

Reply:
Not sure of the easy way besides using node js or uploading all the files to the server :(

Reply:
The issue with hardcoding this is that now you have two places where this information is stored. If you add a new report, you'll need to update both the server and the client.

One way to resolve this is to add a new API endpoint to your server that returns a list of available reports. The client can then just iterate over each report.

Best to do this in a separate PR though.